src/zc/buildout/easy_install.py

```diff
@@ -175,7 +178,7 @@ def _execute_permission():
     return 0o777 - current_umask
 
 
-_easy_install_cmd = 'from setuptools.command.easy_install import main; main()'
+_pip_install_cmd = 'from pip._internal.cli.main import main; main()'
```

Comment:
No protection against pip internal changes here.

Reply:
I should clarify that this is not criticism.  I wanted to note where the dependencies are to see how exposed we would be to pip changing its implementation details.

The index parsing code is protected somewhat (with slight regression in behaviour), the FakeFile import can be replaced, and this one seems very unlikely to change/very easy to fix if changed.

So yay!